tests/unittests/tiglFuselageFrames.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+* Copyright (c) 2020 RISC Software GmbH
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "test.h"
+#include "tigl.h"
+
+#include <Bnd_Box.hxx>
+#include <BRepBndLib.hxx>
+
+#include "CNamedShape.h"
+#include "PNamedShape.h"
+#include "CCPACSConfigurationManager.h"
+#include "CCPACSConfiguration.h"
+#include "CCPACSFrame.h"
+#include "CCPACSFuselage.h"
+
+
+TEST(TestFuselageFrame, bug694)
+{
+    const char* filename = "TestData/bugs/694/test_frame.xml";
+
+    TiglCPACSConfigurationHandle tiglHandle = -1;
+    TixiDocumentHandle tixiHandle = -1;
+
+    ASSERT_EQ(SUCCESS, tixiOpenDocument(filename, &tixiHandle));
```

Comment:
Please close both handles at the end of the test.

Alternatively, use the RAII class ``TiglHandleWrapper`` that does this automatically:
https://github.com/DLR-SC/tigl/blob/master/tests/unittests/test.h#L119

Reply:
Resolved in 1b6ec0f